lib/Target/SubstraitPB/Import.cpp

```diff
@@ -128,6 +163,66 @@ importExpression(ImplicitLocOpBuilder builder, const Expression &message) {
   }
 }
 
+static mlir::FailureOr<FieldReferenceOp>
+importFieldReference(ImplicitLocOpBuilder builder,
+                     const Expression::FieldReference &message) {
+  using ReferenceSegment = Expression::ReferenceSegment;
+
+  MLIRContext *context = builder.getContext();
+  Location loc = UnknownLoc::get(context);
+
+  // Emit error on unsupported cases.
+  // TODO(ingomueller): support more cases.
+  if (!message.has_direct_reference())
+    return emitError(loc) << "only direct reference supported";
+
+  // Traverse list to extract indexes.
+  llvm::SmallVector<int64_t> indexes;
```

Comment:
indices ?

Reply:
(i'm not sure which one is used more db side)

Reply:
This is due to me not being a native speaker, I think. Fixed in #840.